src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -253,231 +261,110 @@ private void CheckAnalysisWasRequested()
 
         #endregion
 
-        #region Processing analysis results tests
```

Comment:
The GitHub diff isn't very helpful here.

All of the tests from this region were moved to the `IssueHandlerTests`.